plenum/server/node.py

```diff
@@ -3246,24 +3286,23 @@ def select_primaries(self, nodeReg: Dict[str, HA] = None):
                 for calculation of primaries for backup instances.
                 '''
                 if instance_id == 0:
-                    primary_rank = self.get_rank_by_name(name, nodeReg)
+                    primary_rank = self.get_rank_by_name(name)
```

Comment:
it used to restore primaries basing on certain nodeReg, particulary it used with CurrentState. Since node.nodeReg consistent with poolLedger txns, in current primary selection implementation, we can just use it.